src/ray/raylet/node_manager.cc

```diff
@@ -3254,22 +3253,8 @@ void NodeManager::HandleGetNodeStats(const rpc::GetNodeStatsRequest &node_stats_
     worker->rpc_client()->GetCoreWorkerStats(
         request, [reply, worker, all_workers, driver_ids, send_reply_callback](
                      const ray::Status &status, const rpc::GetCoreWorkerStatsReply &r) {
-          auto worker_stats = reply->add_workers_stats();
-          worker_stats->set_pid(worker->GetProcess().GetId());
-          worker_stats->set_worker_id(worker->WorkerId().Binary());
-          worker_stats->set_is_driver(driver_ids.contains(worker->WorkerId()));
-          worker_stats->set_language(worker->GetLanguage());
+          reply->add_core_workers_stats()->MergeFrom(r.core_worker_stats());
           reply->set_num_workers(reply->num_workers() + 1);
-          if (status.ok()) {
-            worker_stats->mutable_core_worker_stats()->MergeFrom(r.core_worker_stats());
-          } else {
```

Comment:
why remove this error log?
i encounter `grpc: Sent message too large` error in version 1.0.0, and find it disappear in version 1.0.1

is this error not importent?

Reply:
Dashboard will query every core worker stats periodically, so general errors can be resolved by retrying. However, the `grpc: Sent message too large` error is important, I will add the error message back in next PR.